front/src/templates/DeclarationProductPageMobile.tsx

```
@@ -331,23 +333,26 @@ const Declaration = () => {
+                        country={meansOfTransportAndCountry.country}
+                        border={declarationAgentRequest.border}
+                        onDelete={(id) => {
+                          setDeletedProductId(id);
+                          setOpenModalDeleteProduct(true);
+                        }}
+                        editable={isAvailableToEdit}
+                        onModifyClick={onModifyClick}
+                        key={`${amountProduct.group}-${index}`}
+                      />
+                    );
+                  })}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)